src/pages/AfterLogin/Render.tsx

```diff
@@ -1,55 +1,44 @@
-import Nav from "./Nav";
+import Nav from '../../components/Navbar/Nav';
 
-import React from 'react'
-import SearchInput from "./SearchInput";
-import Cards from "./Cards";
-import Slider from "./Slider";
+import React from 'react';
+import SearchInput from './SearchInput';
+import Cards from './Cards';
+import Slider from './Slider';
 
+const Render = () => {
+  const slides = [
```

Comment:
Maybe we should move these constants outside of these files, just to keep them clutter free, you can create another file somewhere called constants or data or something along those lines and import it from there.